pkg/channel/fanout/fanout_handler.go

```diff
@@ -22,10 +22,11 @@ limitations under the License.
 package fanout
 
 import (
+	"context"
 	"errors"
-	"net/http"
 	"time"
 
+	cloudevents "github.com/cloudevents/sdk-go"
```

Comment:
Goland says: redundant alias :-)

Reply:
yes, I also use Goland. But we have this everywhere, and in many reviews I've been asked to add it back. 
I'm fine with leaving it as is. Let me know if you are not, and I can change it